examples/metrics.py

```diff
@@ -65,8 +76,41 @@ def run(self, run_options: dict) -> None:
                 self.aggregate(repo_node)
                 name = f'https://github.com/{org}/{repo}'
                 self.summarize(name, reporting_period, repo_node)
+
+        # Until we move our entire data to Datadog, we will continue to push data to Google Sheet as well
         self.output_google_sheet(reporting_period=reporting_period)
+        
+        # Convert data to Datadog time series
+        datadog_series = []
+
+        for org in global_node['nodes']:
+            org_node = global_node['nodes'][org]
+            for repo in org_node['nodes']:
+                repo_node = org_node['nodes'][repo]
+                datadog_series += self.get_series_for_datadog(repo_node, org, repo)
 
+        print("Datadog series data:", datadog_series)
+
+        # Submit data to Datadog
+        self.datadog_api.submit_metrics(datadog_series)
+    
+    def get_series_for_datadog(self, repo_node: Dict, org: str, repo: str) -> Iterator[Series]:
+        for metric_name, metric_type in DATADOG_METRICS:
+            
+            try:
+                data = repo_node['metrics'][metric_name][metric_type]
+
+            except KeyError as e:
+                print(f'Failed to find metric "{metric_name}" for {repo}: {e}')
+                continue
+
+            yield Series(
+                metric=f'helper_library.{metric_name}.{metric_type}',
```

Comment:
I think we should make this consistent with the Datadog Release Metric name (`library.release.count`), with an associated tag of `type:helper`,
so in this case the metric name would be `f'library.{metric_name}.{metric_type}'`, and then add `'type:helper'` to the tags in line 111

Reply:
Done.